package.json

```diff
@@ -1,17 +1,26 @@
 {
-  "author": {
-    "name": "Alex Kaul",
-    "email": "alexkaul@googlemail.com"
-  },
-  "name": "ng-img-crop",
-  "prettyName": "ngImgCrop",
-  "version": "0.3.2",
-  "description": "Image crop directive for AngularJS",
+  "author": [
+    {
+      "name": "Alex Kaul",
+      "email": "alexkaul@googlemail.com"
+    },
+    {
+      "name": "Jodonnell-broadsoft"
+    },
+    {
+      "name": "Marcio Sena",
+      "email": "marciosena.inf@gmail.com"
+    }
+  ],
+  "name": "ng-img-crop-add-rectangle",
+  "prettyName": "ngImgCropAddRectangle",
+  "version": "0.3.3",
+  "description": "Image crop directive for AngularJS add rectangle",
   "license": "MIT",
-  "homepage": "https://github.com/alexk111/ngImgCrop",
+  "homepage": "https://github.com/marciovsena/ngImgCrop/",
```

Comment:
https://github.com/marciovsena/ngImgCrop/, version doesn't work and is incomplete in documentation, ie:  area-coords="myAreaCoords"